core/src/main/resources/jenkins/model/Jenkins/legend.jelly

```diff
@@ -23,142 +23,134 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-  <l:layout title="${%Icon legend}">
-    <l:app-bar title="${title}" />
-
-    <st:include page="sidepanel.jelly"/>
-    <l:breadcrumb title="${%title}" />
-
-    <l:main-panel>
-      <h2>${%Status}</h2>
-
-      <dl class="app-icon-legend">
-        <dt>
-          <l:icon class="icon-blue icon-xlg"/>
-        </dt>
-        <dd>
-          ${%blue}
-        </dd>
-
-        <dt>
-          <l:icon class="icon-blue-anime icon-xlg"/>
-        </dt>
-        <dd>
-          ${%blue_anime}
-        </dd>
-
-        <dt>
-          <l:icon class="icon-yellow icon-xlg"/>
-        </dt>
-        <dd>
-          ${%yellow}
-        </dd>
-
-        <dt>
-          <l:icon class="icon-yellow-anime icon-xlg"/>
-        </dt>
-        <dd>
-          ${%yellow_anime}
-        </dd>
-
-        <dt>
-          <l:icon class="icon-red icon-xlg"/>
-        </dt>
-        <dd>
-          ${%red}
-        </dd>
-
-        <dt>
-          <l:icon class="icon-red-anime icon-xlg"/>
-        </dt>
-        <dd>
-          ${%red_anime}
-        </dd>
-
-        <dt>
-          <l:icon class="icon-nobuilt icon-xlg"/>
-        </dt>
-        <dd>
-          ${%lightgrey}
-        </dd>
-
-        <dt>
-          <l:icon class="icon-nobuilt-anime icon-xlg"/>
-        </dt>
-        <dd>
-          ${%lightgrey_anime}
-        </dd>
-
-        <dt>
-          <l:icon class="icon-disabled icon-xlg"/>
-        </dt>
-        <dd>
-          ${%grey}
-        </dd>
-
-        <dt>
-          <l:icon class="icon-disabled-anime icon-xlg"/>
-        </dt>
-        <dd>
-          ${%grey_anime}
-        </dd>
-
-        <dt>
-          <l:icon class="icon-aborted icon-xlg"/>
-        </dt>
-        <dd>
-          ${%darkgrey}
-        </dd>
-
-        <dt>
-          <l:icon class="icon-aborted-anime icon-xlg"/>
-        </dt>
-        <dd>
-          ${%darkgrey_anime}
-        </dd>
-      </dl>
-
-      <h2>${%Project Health}</h2>
-
-      <dl class="app-icon-legend">
-        <dt>
-          <l:icon src="symbol-weather-icon-health-80plus" />
-        </dt>
-        <dd>
-          ${%health-81plus}
-        </dd>
-
-        <dt>
-          <l:icon src="symbol-weather-icon-health-60to79" />
-        </dt>
-        <dd>
-          ${%health-61to80}
-        </dd>
-
-        <dt>
-          <l:icon src="symbol-weather-icon-health-40to59" />
-        </dt>
-        <dd>
-          ${%health-41to60}
-        </dd>
-
-        <dt>
-          <l:icon src="symbol-weather-icon-health-20to39" />
-        </dt>
-        <dd>
-          ${%health-21to40}
-        </dd>
-
-        <dt>
-          <l:icon src="symbol-weather-icon-health-00to19" />
-        </dt>
-        <dd>
-          ${%health-00to20}
-        </dd>
-      </dl>
-    </l:main-panel>
-  </l:layout>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
+  <h1 class="jenkins-modal__title">${%Icon legend}</h1>
```

Comment:
Since this is no longer a standalone view, attempts to access the URL `/legend` (e.g. from links in outside documentation, or browser history) will result in the logging of the message

```
Mar 17, 2023 5:00:06 PM WARNING jenkins.security.stapler.StaplerFilteredActionListener onDispatchTrigger
New Stapler dispatch rules result in the URL "/legend" no longer being allowed. If you consider it safe to use, add the following to the whitelist: "hudson.model.Hudson legend". Learn more: https://www.jenkins.io/redirect/stapler-facet-restrictions
```

This seems unnecessary. Perhaps rename the view to `_legend` so it's a simple 404?

Reply:
https://github.com/jenkinsci/jenkins/pull/7746